ubo-common/src/main/java/org/mycore/ubo/importer/ImportListJobAction.java

```diff
@@ -66,8 +67,15 @@ public void execute() throws ExecutionException {
                 importJob.enrich();
             }
 
-            importJob.saveAndIndex();
-            sendMail(importJob);
+            try {
+                MCRTransactionHelper.beginTransaction();
```

Comment:
The job should already have a transaction. The MCRJobRunnable creates a transaction for every MCRJobAction.